DatadogInternal/Sources/NetworkInstrumentation/URLSession/URLSessionTaskInterception.swift

```diff
@@ -50,6 +50,10 @@ public class URLSessionTaskInterception {
         }
     }
 
+    func register(request: URLRequest) {
+        self.request = request
+    }
```

Comment:
Maybe it's ok to just have var request instead.

Reply:
going with the established pattern like we have `metrics`, `data` etc properties here.